diff --git a/node_modules/svelte-sonner/.bun-tag-b69ffa38702539fc b/.bun-tag-b69ffa38702539fc
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/Toast.svelte b/dist/Toast.svelte
index 7d122ef44fd2a8a17b9c7ccddefd564b76de2260..efdaf97a72bdf3ab3450928e505abfb3036e82be 100644
--- a/dist/Toast.svelte
+++ b/dist/Toast.svelte
@@ -56,7 +56,7 @@ let swipeOut = $state(false);
 let isSwiped = $state(false);
 let offsetBeforeRemove = $state(0);
 let initialHeight = $state(0);
-let remainingTime = toast.duration || durationFromToaster || TOAST_LIFETIME;
+let remainingTime = $state(toast.duration || durationFromToaster || TOAST_LIFETIME);
 let dragStartTime = $state(null);
 let toastRef = $state();
 let swipeDirection = $state(null);
@@ -393,7 +393,7 @@ const icon = $derived.by(() => {
 
 	{#if toast.component}
 		{@const Component = toast.component}
-		<Component {...toast.componentProps} closeToast={deleteToast} remainingTime={remainingTime} />
+		<Component {...toast.componentProps} closeToast={deleteToast} remainingTime={remainingTime} {expanded} />
 	{:else}
 		{#if (toastType || toast.icon || toast.promise) && toast.icon !== null && (icon !== null || toast.icon)}
 			<div data-icon="" class={cn(classes?.icon, toast?.classes?.icon)}>
